inc/fields/actorfield.class.php

```diff
@@ -27,17 +28,17 @@ public function displayField($canEdit = true) {
       // Value needs to be non empty to allow execition of select2's initSelection
       echo '<input
                type="hidden"
-               name="formcreator_field_' . $this->fields['id'] . '"
-               id="actor_formcreator_field_' . $this->fields['id'] . '"
-               value=" " />';
```

Comment:
value=" " was source of problems when Select2 component populates this atribute. IDs of users were prefixed and/or suffixed by a comma leading this kind of result:

,2,
,2,5
2,5,